components/02-molecules/pull-quote/pull-quote.twig

```diff
@@ -0,0 +1,35 @@
+{#
+ # Available Props:
```

Comment:
Missing some variable documentation here but I like the format. Example: `pull_quote__base_class`, `pull_quote__modifiers`, `pull_quote__blockname`,`pull_quote__attribute`

Reply:
For what it's worth, I don't usually include the "class variables" (base_class, modifiers, blockname, and additional_classes) to the docs at the top. From my perspective, they're a given, and it just adds more text to the page. Same would probably go for the attributes...

My preference is to keep the docs at the top specific to "Here are the content variables you can/should provide, and here are the 'props' available to manipulate the variation"

Reply:
@joetower I think @ModulesUnraveled has a valid point. Lets remove lines 8, 9, and 10 from this file and call this PR good to merge